llvm/lib/Target/RISCV/MCTargetDesc/RISCVFixupKinds.h

```diff
@@ -71,6 +71,18 @@ enum Fixups {
   // Used to generate an R_RISCV_ALIGN relocation, which indicates the linker
   // should fixup the alignment after linker relaxation.
   fixup_riscv_align,
+  // 20-bit fixup corresponding to %tlsdesc_hi(foo) for instructions like
+  // auipc
+  fixup_riscv_tlsdesc_hi20,
```

Comment:
> instructions like auipc

It seems that tlsdesc is auipc-style only, instead of lui-style. You can say that this is for auipc.

Reply:
I wonder whether we can just use `MCFixupKind(FirstLiteralRelocationKind + ELF::R_RISCV_TLSDESC_CALL)`

Reply:
I've adjusted the comments per your suggestion, but could you elaborate more on what you're thinking w.r.t. `MCFixupKind(FirstLiteralRelocationKind + ELF::R_RISCV_TLSDESC_CALL)`?